src/Scaffold_Command.php

```diff
@@ -866,6 +968,20 @@ protected function log_whether_files_written( $files_written, $skip_message, $su
 		}
 	}
 
+	/**
+	 * Extract dashicon name when provided or return null otherwise.
+	 *
+	 * @param array $assoc_args
+	 * @return string|null
+	 */
+	private function maybe_extract_dashicon( $assoc_args ) {
```

Comment:
I'm not a huge fan of functions that start with `maybe_` because it implies their behavior is ambiguous. Can we come up with a better name for it?

Reply:
Welcome to the club. I followed the existing code: `maybe_create_themes_dir` and `maybe_create_plugins_dir`. I will figure out something else. I want to avoid code duplication here.